param/obstacle_detector.yaml

```diff
@@ -0,0 +1,32 @@
+# Obstacle detector configuration
+# Everything in mks units, all angles in radians, unless otherwise stated
+
+# Boundaries of the arena
+# Points outside of these boundaries are not considered obstacles
+min_x: -5.0
+max_x: 5.0
+min_y: -5.0
+max_y: 5.0
+min_z: 0.2
+max_z: 4.0
+
+# Points inside a cylinder of this radius are not considered obstacles
+drone_radius: 0.6
+
+# Points this close together or closer are considered part of the same obstacle
+cluster_threshold: 0.2
+
+# Position of the center of the arena
+arena_center_x: 0.0
+arena_center_y: 0.0
+
+# Parameters describing the obstacles
+obst_radius: 0.05
+obst_height: 3.0
```

Comment:
If this is in meters, why so high? Is this for safety reasons now and eventually, when we have depth cameras, we will actual find the heights?

Reply:
Yeah this version of the obstacle detector doesn't have any concept of how tall the obstacles are so I just made them as tall as possible.  Future versions will account for height